### PR TITLE
Update Kubeflow Index Page

### PR DESCRIPTION
**Pull request by andreyvelich**
Followup on @thesuperzapper comment here: https://github.com/kubeflow/website/pull/3728#discussion_r1628261603

I updated Kubeflow index page in this PR. Please let me know what do you think about this statement ?
For inspiration I checked one of the first Kubeflow videos that @aronchick presented before:
https://youtu.be/R3dVF5wWz-g?t=875

/assign @thesuperzapper @kubeflow/kubeflow-steering-committee @StefanoFioravanzo @hbelmiro @franciscojavierarceo 

/hold for review

### CODE REVIEW

File: content/en/_index.html
Comment (hbelmiro) [inline content/en/_index.html:33]:
```suggestion
      The Kubeflow project makes AI/ML on Kubernetes simple, portable and scalable.
      Our goal is not to recreate other services, but to provide a straightforward way to learn,
      deploy, and manage best-in-class open-source systems for AI/ML on any cloud infrastructure.
```

File: content/en/docs/started/introduction.md
Comment (hbelmiro) [inline content/en/docs/started/introduction.md:12]:
```suggestion
Our goal is not to recreate other services, but to provide a straightforward way to learn,
deploy, and manage best-in-class open-source systems for AI/ML on any cloud infrastructure.
```

**Review by hbelmiro — COMMENTED** [review]
/lgtm

I left a few small suggestions though. Totally optional.

**Comment by thesuperzapper** [comment]
@andreyvelich Here are the key points that I think we should hit for the front-page `"What is Kubeflow?"` wording:

1. It should convey what Kubeflow does, and its key value proposition.
2. It should be no longer than 3 sentences.
3. It should make sense to a person in a leadership position (not just technical).
4. It should not use acronyms like AI and ML, but rather their full words.

Here are the key points that I think makes Kubeflow special:

- Kubeflow is for machine learning on Kubernetes.
- Kubeflow is not a single tool, we are an ecosystem of components for each stage in the ML lifecycle.
- It can run on any Kubernetes cluster.

## Based on that, here is my proposed wording:

> Kubeflow enables machine learning on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at any scale. We are an _ecosystem_ of components for each stage in the [machine learning lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle).
>
> Anywhere you run Kubernetes, you should be able to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/).

## Here is with full HTML:

```html
Kubeflow enables machine learning on <a href="https://kubernetes.io/docs/concepts/overview/" target="_blank" rel="noopener">Kubernetes</a> at any scale.
We are an <em>ecosystem</em> of components for each stage in the <a href="/docs/started/architecture/#introducing-the-ml-lifecycle" target="_blank" rel="noopener">machine learning lifecycle</a>.
<br><br>
Anywhere you run Kubernetes, you should be able to <a href="/docs/started/installing-kubeflow/" target="_blank" rel="noopener">install Kubeflow</a>.
```

**Comment by andreyvelich** [comment]
Thank you for proposing this @thesuperzapper, I agree with some statements.
However, don't you think that we should clearly say that Kubeflow is a bridge between ML and Cloud ecosystems ? We allow users to leverage cloud resources to scale their ML development and AI applications.

Also, we might need to emphasize that Kubeflow is also for artificial intelligence not only for machine learning. For example, Kubeflow components are enabled AI running on the cloud. 

Any other thoughts from the community @kubeflow/wg-data-leads @kubeflow/wg-pipeline-leads @kubeflow/wg-training-leads @kubeflow/wg-notebooks-leads @kubeflow/wg-manifests-leads @vikas-saxena02 @hbelmiro ?

**Comment by tarilabs** [comment]
proposal iterating from Mathew's suggestion:

> Kubeflow enables machine learning on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at scale. Kubeflow is an ecosystem of components for each stage in the [machine learning lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle), tuned and orchestrated to work in [cloud-native](https://github.com/cncf/toc/blob/main/DEFINITION.md) way.
> 
> Anywhere you run Kubernetes, you should be able to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/).

I also like the statement about not-recreate components.

**Comment by juliusvonkohout** [comment]
@thesuperzapper 

 i would like to add to:

Here are the key points that I think makes Kubeflow special:

- Kubeflow is for machine learning on Kubernetes.
- Kubeflow is not a single tool, we are an ecosystem of components for each stage in the ML lifecycle.
- It can run on any Kubernetes cluster.
- **It is open source and can be modified and extended as needed**
- **There is no vendor lock-in**

**Comment by StefanoFioravanzo** [comment]
I like where @thesuperzapper is going with this.

On @andreyvelich 's comments:

> However, don't you think that we should clearly say that Kubeflow is a bridge between ML and Cloud ecosystems ? We allow users to leverage cloud resources to scale their ML development and AI applications. For example, if folks want to run PyTorch on a cloud, they need to come to us.

You feel like the second sentence does not emphasize this enough?

> Also, we might need to emphasize that Kubeflow is also for artificial intelligence not only for machine learning. For example, Kubeflow components are enabled AI running on the cloud.

Do you feel like we should word it as "Kubeflow enables Artificial Intelligence and Machine Learning on Kubernetes ..." ?
With the goal of keeping our statement short and clear, what value does this bring to the message we want to convey?

**Comment by andreyvelich** [comment]
>You feel like the second sentence does not emphasize this enough?

@StefanoFioravanzo Which sentence ?

>With the goal of keeping our statement short and clear, what value does this bring to the message we want to convey?

That shows the value of Kubeflow to develop AI applications.

**Comment by vikas-saxena02** [comment]
The proposed changes looks good to me

**Comment by thesuperzapper** [comment]
@vikas-saxena02 which proposal are you supporting?

Also, for others, let's continue discussing so we can converge on a wording (rather than LGTM the PR). 

@andreyvelich perhaps you can mark this PR as a draft to indicate that we are still discussing.

**Comment by thesuperzapper** [comment]
As I was saying in https://github.com/kubeflow/website/pull/3755#issuecomment-2163722592, the core purpose of the intro paragraph is to help users learn about what Kubeflow is.

I am happy for people to suggest changes to the proposal I made:

> Kubeflow enables machine learning on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at any scale. We are an _ecosystem_ of components for each stage in the [machine learning lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle).
>
> Anywhere you run Kubernetes, you should be able to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/).

---

Regarding the specific comments from @andreyvelich:

> However, don't you think that we should clearly say that Kubeflow is a bridge between ML and Cloud ecosystems ? We allow users to leverage cloud resources to scale their ML development and AI applications. For example, if folks want to run PyTorch on a cloud, they need to come to us.

I think that message is already conveyed by the first paragraph, is there a specific wording change you are suggesting? 

Also, "Kubernetes" is clearer than the word "Cloud" because Kubernetes can be run anywhere, and cloud is not really defined. 

> Also, we might need to emphasize that Kubeflow is also for artificial intelligence not only for machine learning. For example, Kubeflow components are enabled AI running on the cloud.

I don't think adding the words "Artificial Intelligence" will add any significant value and may actually detract from the simplicity of the wording. Furthermore, it's a largely undefined term that has lots of emotion associated to it right now.

**Comment by StefanoFioravanzo** [comment]
I agree with @thesuperzapper  and I think the wording he proposed is already a great improvement over what we have now. I share your concerns on the use of "Artificial Intelligence".

If the intent of adding "Artificial Intelligence" is to hint at support for GenAI / LLM applications and use cases, I suggest thinking about a new section in the index page, maybe under components, that highlights supported use cases.

**Comment by andreyvelich** [comment]
@StefanoFioravanzo @thesuperzapper Why we don't want to add acronyms in our index page ?
We can always explain what those acronyms mean in the [introduction page](https://www.kubeflow.org/docs/started/introduction/#what-is-kubeflow).
I think, AI/ML is the well-known statement today, and users who want to explore Kubeflow might already aware of AI and ML. Especially, with the existing value that Kubeflow brings for GenAI and LLM applications.

---
For the first statement, why you don't like this statement ?
```
The Kubeflow makes AI/ML on Kubernetes simple, portable and scalable.
```

---

For the second, I like what @thesuperzapper proposed with this modification:
```
We are an ecosystem of projects for each stage in the ML lifecycle
```

**Review by terrytangyuan — COMMENTED** [review]
I am okay with the use of "AI/ML" here for simplicity. They are very widely used terms nowadays. Spelling them out looks a bit mouthful to me.

**Comment by thesuperzapper** [comment]
My vote is just to say "machine learning" as it comprehensively describes the purpose of Kubeflow. 

I don't think "artificial intelligence" is a usefully defined term at this point (and even less so "AI"), it's really just a buzzword, and using it makes us subject to confusion as the term evolves. 

---

My proposal is still to use something like was proposed in https://github.com/kubeflow/website/pull/3755#issuecomment-2163722592.

Are there any specific suggestions people have for the following introduction language:

> Kubeflow enables machine learning on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at any scale. We are an _ecosystem_ of components for each stage in the [machine learning lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle).
>
> Anywhere you run Kubernetes, you should be able to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/).

**Comment by vikas-saxena02** [comment]
> My vote is just to say "machine learning" as it comprehensively describes the purpose of Kubeflow.
> 
> I don't think "artificial intelligence" is a usefully defined term at this point (and even less so "AI"), it's really just a buzzword, and using it makes us subject to confusion as the term evolves.
> 
> My proposal is still to use something like was proposed in [#3755 (comment)](https://github.com/kubeflow/website/pull/3755#issuecomment-2163722592).
> 
> Are there any specific suggestions people have for the following introduction language:
> 
> > Kubeflow enables machine learning on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at any scale. We are an _ecosystem_ of components for each stage in the [machine learning lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle).
> > Anywhere you run Kubernetes, you should be able to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/).

I agree with @thesuperzapper above as ML and AI are kind of synonymous but ML is more suited in Kubeflow context as its more appropriate and followed in literature. AI is more or a less buzzword as @thesuperzapper suggested.

**Comment by franciscojavierarceo** [comment]
My two cents here is that "AI" is useful for SEO and the uninitiated. Maybe our plan is to stay narrow and then the SEO piece doesn't matter.

For Feast, I hope to expand the target audience.

**Comment by andreyvelich** [comment]
What is your main concern with AI in the index page @thesuperzapper ?

Look at the other communities, they do the same today: [Metaflow](https://metaflow.org/), [MLFlow](https://mlflow.org/), [Ray](https://www.ray.io/).
It helps us to explain that Kubeflow is the main place for users/developers/researchers for AI and ML on Kubernetes. Some folks hear about AI some folks hear about ML. We want to attract all of them.
I believe, it should help us to get more interest in Kubeflow ecosystem from the new users.

Additionally, it help us to be closer to CNCF glossary:  e.g. CNAI Landscape and CNCF AI WG: https://landscape.cncf.io/?group=cnai 
cc @ronaldpetty @raravena80 @zanetworker

**Review by terrytangyuan — APPROVED** [review]
/lgtm

**Comment by thesuperzapper** [comment]
@terrytangyuan I think there was more agreement with my wording than the one @andreyvelich proposed.

Personally I still don't like the current proposal after the force push, and I think we should use something like this:

> Kubeflow enables machine learning on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at any scale. We are an _ecosystem_ of components for each stage in the [machine learning lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle).
>
> Anywhere you run Kubernetes, you should be able to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/).

It's much more punchy and actually hits the points of what Kubeflow is, see https://github.com/kubeflow/website/pull/3755#issuecomment-2163722592 for an explanation of how I came up with my proposal.

**Comment by andreyvelich** [comment]
@thesuperzapper This PR has been opened for 1.5 months, and I didn't get any feedback after this comment: https://github.com/kubeflow/website/pull/3755#issuecomment-2209512860, so we decided to move this forward.

A few points:
- I think, we should include AI in our description as I mentioned here: https://github.com/kubeflow/website/pull/3755#issuecomment-2209512860. Also @terrytangyuan and @franciscojavierarceo has similar thoughts
- Using Kubeflow is not only about scale, but also about experience, simplicity, and portability. That is why I say:
 ` AI/ML on Kubernetes simple, portable and scalable.`
- Additionally, the second message clearly say that Kubeflow is bridge between ML and Cloud Ecosystems.

**Comment by franciscojavierarceo** [comment]
I strongly believe we should use AI/ML as it will be useful for SEO and helping new users understand that they're really quite similar. By not including AI we are explicitly forgoing engaging more potential users of Kubeflow and I don't think that's what we want.

While I do agree with @thesuperzapper that acronyms are bad, so long as we define them initially, that should be sufficient.

**Comment by thesuperzapper** [comment]
@andreyvelich @franciscojavierarceo   I think the currently proposed text fails to say the most important thing, which is that Kubeflow is an __ecosystem of components__ that run on Kubernetes.

If others want to include artificial intelligence, that's fine, but let's use its full name to avoid ambiguity.
Here is a version of my proposal incorporating all the suggestions: 

> Kubeflow enables artificial intelligence and machine learning on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at any scale. We are an _ecosystem_ of portable components for each stage in the [machine learning lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle).
>
> Anywhere you run Kubernetes, you should be able to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/).

If you agree, also include the links, as they help users find out more about specific things like _"machine learning lifecycle",  _"Kubernetes"_ and _"installing kubeflow"_.

---

For context. here is the current proposal from the PR:

> The Kubeflow project makes AI/ML on Kubernetes simple, portable and scalable.
> Our goal is to provide a straightforward way to learn, deploy, and manage best-in-class open-source systems for AI/ML on any cloud infrastructure.
>
> Anywhere you are running Kubernetes, you should be able to run Kubeflow.

**Comment by franciscojavierarceo** [comment]
Yeah, I am good with that 👍 . My only nit is that I do think we should try to add "AI/ML" in different places to explicitly optimize the pages for SEO.

**Comment by andreyvelich** [comment]
Alright, what do we think about this:

> The Kubeflow project enables artificial intelligence (AI) and machine learning (ML) on [Kubernetes](https://kubernetes.io/) at any scale. We are an *ecosystem* of portable components for each stage in the [ML lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle).
> 
> Our goal is to provide a straightforward way to learn, deploy, and manage best-in-class [open-source systems](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem) for AI/ML on any cloud infrastructure.
>
> Anywhere you run Kubernetes, you should be able to [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/).

**Comment by thesuperzapper** [comment]
@andreyvelich we can just say `Kubeflow` at the beginning, and possibly combine the last two sentences as they kind of say the same thing.

What about this:

> Kubeflow enables artificial intelligence (AI) and machine learning (ML) on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at any scale. We are an _ecosystem_ of portable components for each stage in the [AI/ML Lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle).
>
> Anywhere you run Kubernetes, you can [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/) to create an open-source AI/ML platform.

Also, getting the word "platform" in is good for SEO.

**Comment by andreyvelich** [comment]
The main message I am trying to tell from the 3rd sentence is that we are not re-creating the ML frameworks, but we are providing a way to run them on Kubernetes in a right way.
Which should tell users that if they want to run AI/ML on the cloud, they have to use Kubeflow.

**Comment by thesuperzapper** [comment]
@andreyvelich I did not understand what you were trying to say. We can avoid someone thinking we are competing with PyTorch by adding `with integrations for all major frameworks` to the end of the second sentence. 

For example, what about this:

> Kubeflow enables artificial intelligence (AI) and machine learning (ML) on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) at any scale. We are an _ecosystem_ of portable components for each stage in the [AI/ML Lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle) with integrations for all major frameworks.
>
> Anywhere you run Kubernetes, you can [install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/) to create an open-source AI/ML platform.

**Comment by andreyvelich** [comment]
Ok, why you don't like the first message as this:
> Kubeflow makes artificial intelligence (AI) and machine learning (ML) on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) simple, portable, and scalable.

I believe, it is important to say about simplicity, right now your message doesn't say that.

As I mentioned before that was one of the key points when Kubeflow was introduced in 2017: https://youtu.be/R3dVF5wWz-g?t=884

**Comment by thesuperzapper** [comment]
> Ok, why you don't like the first message as this:
> 
> > Kubeflow makes artificial intelligence (AI) and machine learning (ML) on [Kubernetes](https://kubernetes.io/docs/concepts/overview/) simple, portable, and scalable.
> 
> I believe, it is important to say about simplicity, right now your message doesn't say that.
> 
> As I mentioned before that was one of the key points when Kubeflow was introduced in 2017: https://youtu.be/R3dVF5wWz-g?t=884

@andreyvelich I am not against changing the first sentence, but my aversion to that specific proposal has three causes:

1. The `"simple, portable, and scalable"` wording is most applicable to Kubeflow Pipelines (which what that video was largely introducing), and makes it harder for people to realize that Kubeflow is more than just pipelines.
2. Using the word `"simple"` is subjective at best, and offensive at worst. It's also not really one of the selling points of Kubeflow. People use other platforms if they are looking for a simple solution, they use Kubeflow because it's the only serious option for Kubernetes.

**Comment by andreyvelich** [comment]
>The "simple, portable, and scalable" wording is most applicable to Kubeflow Pipelines (which what that video was largely introducing), and makes it harder for people to realize that Kubeflow is more than just pipelines.

Why was it introduction of Kubeflow Pipelines ? Initially, Kubeflow supports JupyterHub + TensorFlow operator, and the demo was showing how to train + serve model on Kubernetes.

Simplicity, Portability, and Scalability doesn't apply only for KFP. It affects other parts of Kubeflow projects. For example, Training Operator is used to train LLMs given the fault tolerance and advanced scheduling capabilities. Also, with SDK it is very simple to create training jobs. Similar things apply for KServe for model serving, and Katib for HP tuning.
cc @kubeflow/wg-training-leads

**Comment by thesuperzapper** [comment]
@andreyvelich ok, I agree your one sounds better, let's use your sentence as the beginning with a slight variation.

I also think is fine to remove the `(AI)` and `(ML)` brackets, because it looks weird and breaks the flow.

That would leave us with something like this:

> Kubeflow makes artificial intelligence and machine learning simple, portable, and scalable. We are an _ecosystem_ of [Kubernetes](https://kubernetes.io/docs/concepts/overview/) based components for the [AI/ML Lifecycle](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle) with support for all major frameworks.
>
> [Install Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/) anywhere you run Kubernetes.

**Comment by franciscojavierarceo** [comment]
Small suggestion:

> [Run Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/) anywhere you run Kubernetes.

Maybe we want to be explicit with the install but I would generally assume I have to install it, just a thought.

**Comment by andreyvelich** [comment]
Ok, I updated it. What do you think about this message:

> Kubeflow makes artificial intelligence and machine learning simple, portable, and scalable.
> We are an ecosystem of [Kubernetes](https://kubernetes.io/) based components for each stage in [the AI/ML Lifecycle](https://deploy-preview-3755--competent-brattain-de2d6d.netlify.app/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle) with support for best-in-class open source [systems for AI/ML](https://deploy-preview-3755--competent-brattain-de2d6d.netlify.app/docs/started/architecture/#kubeflow-ecosystem).
> 
> Run [Kubeflow](https://deploy-preview-3755--competent-brattain-de2d6d.netlify.app/docs/started/installing-kubeflow/) anywhere you run Kubernetes.

**Comment by thesuperzapper** [comment]
@andreyvelich think it's close, but. I don't like the repetition of "Run", and I think the second sentence is a little cumbersome.

How about:

> Kubeflow makes artificial intelligence and machine learning simple, portable, and scalable.
> We are an ecosystem of [Kubernetes](https://kubernetes.io/) based components for each stage in [the AI/ML Lifecycle](https://deploy-preview-3755--competent-brattain-de2d6d.netlify.app/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle) with support for [best-in-class tools and frameworks](https://deploy-preview-3755--competent-brattain-de2d6d.netlify.app/docs/started/architecture/#kubeflow-ecosystem).
>
> [Deploy Kubeflow](https://deploy-preview-3755--competent-brattain-de2d6d.netlify.app/docs/started/installing-kubeflow/) anywhere you run Kubernetes.

Also, make the link around the full "Deploy Kubeflow" so it's clear what that link is about

**Comment by andreyvelich** [comment]
@thesuperzapper Ok, maybe we should say `open source tools and frameworks`.

File: content/en/docs/started/introduction.md
Comment (thesuperzapper) [inline content/en/docs/started/introduction.md:13]: I think we can remove the "We are..." sentence, because it effectively repeats the first sentence.

File: content/en/_index.html
Comment (thesuperzapper) [inline content/en/_index.html:37]: I think we should move the "Deploy" within the link so it's clear what the link is.

**Review by thesuperzapper — CHANGES_REQUESTED** [review]
@andreyvelich I am happy to LGTM once these small style changes are made, but the overall content is good now.

File: content/en/_index.html
Comment (andreyvelich) [reply content/en/_index.html]: Sure

**Comment by thesuperzapper** [comment]
@andreyvelich thanks for your work on this, I think we are in a pretty good place, and am happy to merge it 

/lgtm

**Review by andreyvelich — COMMENTED** [review]
Thanks everyone for your review!
/hold cancel

**Comment by jbottum** [comment]
Thanks for completing this.  I appreciate that finding the proper words can be challenging.   In the future, we might avoid sentences with multiple preposition phrases  i.e. We are an _ecosystem_ **of Kubernetes
based components for each stage in the ML Lifecycle with support for best-in-class
open source tools and frameworks**, might be something like, The Kubeflow community develops best-in-class, open-source Kubernetes components that streamline the ML lifecycle and simplify the integration of leading tools and frameworks.

**Comment by thesuperzapper** [comment]
> Thanks for completing this.  I appreciate that finding the proper words can be challenging.   In the future, we might avoid sentences with multiple prepositional phrases  i.e. We are an _ecosystem_ **of Kubernetes
> based components for each stage in the ML Lifecycle with support for best-in-class
> open source tools and frameworks**, might be something like, The Kubeflow community develops best-in-class, open-source Kubernetes components that streamline the ML lifecycle and simplify the integration of leading tools and frameworks.

I think those sentences don't actually mean the same thing.

Specifically, we are not claiming that Kubeflow's tools are "best in class" but that Kubeflow tools support the best open frameworks (e.g. PyTorch, TensorFlow, Spark, ...).

**Comment by andreyvelich** [comment]
That's right. We are explaining that Kubeflow deploys those best-in-class OSS tools.
@jbottum Do you feel that we should change the 2nd sentence as follows ?

> The Kubeflow community develop [Kubernetes](https://kubernetes.io/) based components for each stage in [the AI/ML Lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle) with support for best-in-class open source [tools and frameworks](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem).

**Comment by thesuperzapper** [comment]
> That's right. We are explaining that Kubeflow deploys those best-in-class OSS tools.
> @jbottum Do you feel that we should change the 2nd sentence as follows ?
> 
> > The Kubeflow community develop [Kubernetes](https://kubernetes.io/) based components for each stage in [the AI/ML Lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle) with support for best-in-class open source [tools and frameworks](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem).
> 

@andreyvelich what we have now is more inclusive of tools which are part of the "ecosystem" but not technically developed by the Kubeflow community.

Also, I really think the current "We are an ecosystem..." more correctly explains what Kubeflow is, and helps people quickly understand that we're not just one tool.

**Pull request merged**
